asset_mappr/settings.py

```diff
@@ -20,70 +21,78 @@
 # See https://docs.djangoproject.com/en/4.2/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = 'django-insecure-7ohn3+896-evxn%dr+!4e@fnf+zqg%9nnd$mdg4ojdg9zafrsq'
+SECRET_KEY = "django-insecure-7ohn3+896-evxn%dr+!4e@fnf+zqg%9nnd$mdg4ojdg9zafrsq"
```

Comment:
Secret must not be included here. Need to change and take as an environment variable